xml/System.Windows.Annotations/AnnotationHelper.xml

```diff
@@ -308,7 +308,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- <xref:System.Windows.Annotations.AnnotationHelper.GetAnchorInfo%2A> is introduced in the [!INCLUDE[TLA#tla_netframewk](~/includes/tlasharptla-netframewk-md.md)] version 3.5. For more information, see [Versions and Dependencies](~/docs/framework/migration-guide/versions-and-dependencies.md).  
+ <xref:System.Windows.Annotations.AnnotationHelper.GetAnchorInfo%2A> is available starting with introduced in the .NET Framework version 3.5. For more information, see [Versions and Dependencies](~/docs/framework/migration-guide/versions-and-dependencies.md).  
```

Comment:
New sentence reads a bit awkward. But do we need these remarks at all? Once we add all versions, they should be able to see when it was introduced.